Exec Doc 2024-08-22 (#2721)
makerdao/community: governance/votes/Executive vote - August 22, 2024.md

Reviewer: # Spell Sheet Creation

## Pre-Meeting Tasks (Completed by EOD Monday before Spell Meeting)

- [x] Reach out to Ecosystem Actors about possible Spell Contents.
- [x] Inform relevant parties about deadlines (see [Spell Coordination Schedule](https://github.com/makerdao/pe-checklists/blob/master/spell/spell-crafter-mainnet-workflow.md#spell-coordination-schedule)).
- [x] Review recent [polls](https://vote.makerdao.com/polling) for to-do items.
- [x] Review the latest [forum posts](https://forum.makerdao.com/latest) for requested items.

## GovOps Meeting Tasks (Tuesdays, one week before Spell release)

- [x] Obtain roles for this week:
    - [x] Spell Crafter
    - [x] Reviewer (Dewiz)
    - [x] Reviewer (Sidestream)
- [x] Review Requested Contents from Ecosystem Actors.
- [x] Obtain consensus on Spell Items.
- [x] Confirm Repeating Checklist Items:
    - [x] Office Hours (Yes/No)
    - [x] Global Line Modifier (Yes/No)
    - [x] Order of Operations Issues (Yes/No)

## Sheet Creation (Tuesdays, one week before Spell release)

- [x] Duplicate "Executive Contents (Template)" in [Spell Sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?usp=sharing).
- [x] Add a new column with the spell date in the "Spell Progress Tracking" sheet and ensure the progress is kept up to date. *(In Progress, updated until the current step of the process)*
- [x] Rename duplicate tab for expected spell date (in yyyy-mm-dd format).
- [x] Add role and recurring items info from GovOps Meeting.
- [x] Add Spell Items as formatted in the sheet. 
- [x] Complete and confirm checksums for DAI and MKR transfers.
- [ ] SubDAO Spell Crafter delivered the SubDAO spell address and spell PR in Discord with no differences in the code. *(Not applicable)*

## Obtain Confirmations for listed items (Wednesday, confirmations due EOD Friday) 

- [x] Reach out to relevant Ecosystem Actors for item-specific confirmations.
- [x] Correct any discrepancies.
- [x] Confirm items for the `Governance Confirm` column.

---
title: Template - [Executive Vote] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024

> summary: Update PSM State Variable in Conduit contracts to MCD_LITE_PSM_USDC_A, USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A, Update PSM-USDC-A Fees, Update PSM-USDC-A DC-IAM, Update MCD_LITE_PSM_USDC_A buf, Update LITE-PSM-USDC-A DC-IAM, GSM Delay Update, Update LitePSM Keeper Network Job.

Reviewer 2: Inconsistent capitals

```suggestion
summary: Update PSM State Variable in conduit contracts to MCD_LITE_PSM_USDC_A, USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A, update PSM-USDC-A Fees, update PSM-USDC-A DC-IAM, update MCD_LITE_PSM_USDC_A buf, update LITE-PSM-USDC-A DC-IAM, GSM Delay update, update LitePSM Keeper Network Job.
```

> summary: Update PSM State Variable in conduit contracts to MCD_LITE_PSM_USDC_A, USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A, update PSM-USDC-A Fees, update PSM-USDC-A DC-IAM, update MCD_LITE_PSM_USDC_A buf, update LITE-PSM-USDC-A DC-IAM, GSM Delay update, update LitePSM Keeper Network Job.

Reviewer 3: ```suggestion
summary: Update PSM State Variable in conduit contracts to MCD_LITE_PSM_USDC_A, USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A, update PSM-USDC-A Fees, update PSM-USDC-A DC-IAM parameters, update MCD_LITE_PSM_USDC_A buf, update LITE-PSM-USDC-A DC-IAM parameters, GSM Delay update, update LitePSM Keeper Network Job.
```

> date: $2024-08-22T00:00:00.000Z

Reviewer 2: ```suggestion
date: 2024-08-22T00:00:00.000Z
```

address: "$spell_address"

---
# [Executive Proposal] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024

The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Maker Protocol:

> - The PSM state variables in the conduit contracts will be updated to the MCD_LITE_PSM_USDC_A.

Reviewer 2: ```suggestion
- The PSM state variables in the conduit contracts will be updated to MCD_LITE_PSM_USDC_A.
```

> - **200 million USDC** reserves in PSM-USDC-A will be migrated to LITE-PSM-USDC-A.

Reviewer 2: This is not the proposed action - https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839

> Migrate all but 200M USDC reserves from PSM-USDC-A to LitePSM with a script executed in the spell (has to be done before PSM-USDC-A tin & tout is increased in the spell transaction)

> - **200 million USDC** reserves in PSM-USDC-A will be migrated to LITE-PSM-USDC-A.

Reviewer 3: ```suggestion
- Migrate **all but 200M USDC reserves** from PSM-USDC-A to LITE-PSM-USDC-A.
```

- PSM-USDC-A fees will be updated.

> - PSM-USDC-A DC-IAM will be updated.

Reviewer 2: ```suggestion
- PSM-USDC-A DC-IAM parameters will be updated.
```

> - MCD_LITE_PSM_USDC_A `buf` will be updated.

Reviewer 2: ```suggestion
- MCD_LITE_PSM_USDC_A `buf` will be increased.
```

> - MCD_LITE_PSM_USDC_A DC-IAM will be updated.

Reviewer 2: ```suggestion
- MCD_LITE_PSM_USDC_A DC-IAM parameters will be updated.
```

> - The GSM Delay will be updated.

Reviewer 2: ```suggestion
- The GSM Delay will be increased.
```

- [RWA007_A_JAR_INPUT_CONDUIT](https://etherscan.io/address/0xc8bb4e2b249703640e89265e2ae7c9d5ea2af742)
- [RWA007_A_INPUT_CONDUIT](https://etherscan.io/address/0x58f5e979ef74b60a9e5f955553ab8e0e65ba89c9)
- [RWA007_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x701c3a384c613157bf473152844f368f2d6ef191)
- [RWA015_A_INPUT_CONDUIT_JAR_USDC](https://etherscan.io/address/0xb9373c557f3ae8cddd068c1644ed226cfb18a997)
- [RWA015_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0xe08cb5e24862ea86328295d5e5c08972203c20d8)
- [RWA015_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x1e86cb085f249772f7e7443631a87c6bdba2aceb)
- [RWA009_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0x08012ec53a7fabf6f33318dfb93c1289886ebbe1)

### Migrate Reserves from PSM-USDC-A to LITE-PSM-USDC-A

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

> If this executive proposal passes, then the remaining **200 million USDC** in reserve in PSM-USDC-A will be migrated to LITE-PSM-USDC-A.

Reviewer 2: See comment on summary

> If this executive proposal passes, then the remaining **200 million USDC** in reserve in PSM-USDC-A will be migrated to LITE-PSM-USDC-A.

Reviewer 3: ```suggestion
If this executive proposal passes, **all but 200M USDC reserves** from PSM-USDC-A will be migrated to LITE-PSM-USDC-A.
```

### Update PSM-USDC-A Fees

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then PSM-USDC-A fees will be updated as follows:

- PSM-USDC-A [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) will be increased by 0.01 percentage points from 0% to **0.01%**.
- PSM-USDC-A [Fee Out (`tout`)](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) will be increased by 0.01 percentage points from 0% to **0.01%**.

> ### Update PSM-USDC-A DC-IAM

Reviewer 2: ```suggestion
### Update PSM-USDC-A DC-IAM Parameters
```

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then the PSM-USDC-A DC-IAM parameters below will be updated as follows:

- PSM-USDC-DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be decreased by 7,500 million DAI from 10,000 million DAI to **2,500 million DAI**.
- PSM-USDC-A DC-IAM [Target Available Debt (`gap`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) will be decreased by 180 million DAI from 380 million DAI to **200 million DAI**.

> - PSM-USDC-A DC-IAM [Ceiling Increase Cooldown (`ttl`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-) will remain unchanged at **12 hours**.

Reviewer 2: Why aren't we using billion here and in the sheet? This breaks convention.

Reviewer 2: Discussed on Discord - non-blocking but we do not necessarily agree with this change

### Update MCD_LITE_PSM_USDC_A `buf`

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then the MCD_LITE_PSM_USDC_A fixed-sized amount of pre-minted Dai will be increased as follows:

> - [Fixed-sized Amount of Pre-minted Dai (`buf`)](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512#lite-psm-usdc-a-parameters-overview-8) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.

Reviewer 3: ```suggestion
- MCD_LITE_PSM_USDC_A [Fixed-sized Amount of Pre-minted Dai (`buf`)](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512#lite-psm-usdc-a-parameters-overview-8) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.
```

> ### Update LITE-PSM-USDC-A DC-IAM

Reviewer 2: ```suggestion
### Update LITE-PSM-USDC-A DC-IAM Parameters
```

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then the LITE-PSM-USDC-A DC-IAM will be updated as follows:

> - LITE-PSM-USDC-DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be incresed by 7,450 million DAI from 50 million DAI to **7,500 million DAI**.

Reviewer 3: ```suggestion
- LITE-PSM-USDC-DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased by 7,450 million DAI from 50 million DAI to **7,500 million DAI**.
```

- LITE-PSM-USDC-A DC-IAM [Target Available Debt (`gap`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.
- LITE-PSM-USDC-A DC-IAM [Ceiling Increase Cooldown (`ttl`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-) will remain unchanged at **12 hours**.

### Update GSM Delay

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, the [GSM Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be updated as follows:

- The GSM Pause Delay will be increased by 14 hours from 16 hours to **30 hours**.

> ### Update LitePSM Keeper Network Job

Reviewer 2: Consider linking - https://forum.makerdao.com/t/litepsm-job-post-mortem/24841

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then the LitePSM Keeper Network Job will be updated with the following contract calls and parameters as a consequence of the LITE-PSM-USDC-A DC-IAM and `buf` parameter changes described earlier in this document:

> - The old LitePSMJob will be removed from the CronSequencer through contract call `sequencer.removejob(0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be)`.

Reviewer 3: ```suggestion
- The old LitePSM job will be removed from the CronSequencer through contract call `sequencer.removejob(0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be)`.
```

> - The new LitePSMJob will be added to the CronSequencer through contract call `sequencer.addjob(0x0c86162ba3e507592fc8282b07cf18c7f902c401)`.

Reviewer 3: ```suggestion
- The new LitePSM job will be added to the CronSequencer through contract call `sequencer.addjob(0x0c86162ba3e507592fc8282b07cf18c7f902c401)`.
```

- [`fill`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [rushThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be set to **20 million DAI**.
- [`trim`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [gushThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be set to **20 million DAI**.
- [`chug`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [cutThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be remain unchanged at **300,000 DAI**.
- The chainlog entry for [CRON_LITE_PSM_JOB](https://etherscan.io/address/0x689ce517a4dfcf0c5ec466f2757d324fc292c8be) will be updated to [0x0c86162ba3e507592fc8282b07cf18c7f902c401](https://etherscan.io/address/0x0c86162ba3e507592fc8282b07cf18c7f902c401).

> 

Reviewer 3: ```suggestion

The new LitePSM job meets all the requirements outlined in the [LitePSM Job Post-Mortem](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841).

```

---
title: Template - [Executive Vote] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024
summary: Update PSM State Variable in conduit contracts to MCD_LITE_PSM_USDC_A, USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A, update PSM-USDC-A Fees, update PSM-USDC-A DC-IAM parameters, update MCD_LITE_PSM_USDC_A buf, update LITE-PSM-USDC-A DC-IAM parameters, GSM Delay update, update LitePSM Keeper Network Job.
date: 2024-08-22T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024

The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Maker Protocol:

- The PSM state variables in the conduit contracts will be updated to MCD_LITE_PSM_USDC_A.

> - Migrate **all but 200 million USDC reserves** from PSM-USDC-A to LITE-PSM-USDC-A.

Reviewer: ```suggestion
- **All but 200 million USDC reserves** will be migrated from PSM-USDC-A to LITE-PSM-USDC-A.
```

- PSM-USDC-A fees will be updated.
- PSM-USDC-A DC-IAM parameters will be updated.
- MCD_LITE_PSM_USDC_A `buf` will be increased.

> - MCD_LITE_PSM_USDC_A DC-IAM parameters will be updated.

Reviewer 4: ```suggestion
- LITE-PSM-USDC-A DC-IAM parameters will be updated.
```

Reviewer 3: @blimpa, since you used PSM-USDC-A for the old PSM instead of MCD_PSM_USDC_A, I’ll apply this suggestion for consistency

- [RWA007_A_JAR_INPUT_CONDUIT](https://etherscan.io/address/0xc8bb4e2b249703640e89265e2ae7c9d5ea2af742)
- [RWA007_A_INPUT_CONDUIT](https://etherscan.io/address/0x58f5e979ef74b60a9e5f955553ab8e0e65ba89c9)
- [RWA007_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x701c3a384c613157bf473152844f368f2d6ef191)
- [RWA015_A_INPUT_CONDUIT_JAR_USDC](https://etherscan.io/address/0xb9373c557f3ae8cddd068c1644ed226cfb18a997)
- [RWA015_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0xe08cb5e24862ea86328295d5e5c08972203c20d8)
- [RWA015_A_OUTPUT_CONDUIT](https://etherscan.io/address/0x1e86cb085f249772f7e7443631a87c6bdba2aceb)
- [RWA009_A_INPUT_CONDUIT_URN_USDC](https://etherscan.io/address/0x08012ec53a7fabf6f33318dfb93c1289886ebbe1)

### Migrate Reserves from PSM-USDC-A to LITE-PSM-USDC-A

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

> If this executive proposal passes, **all but 200 million USDC reserves** from PSM-USDC-A will be migrated to LITE-PSM-USDC-A.

Reviewer: ```suggestion
If this executive proposal passes, **all but 200 million USDC reserves** will be migrated from the PSM-USDC-A to the LITE-PSM-USDC-A.
```

Reviewer 2: Disagree on this suggestion. We don't think you need articles when referring to the PSMs using their ilk names.

Reviewer 2: Note our approach is consistent with the phrasing used by the technical dev and risk teams in their communications. see: https://forum.makerdao.com/t/litepsm-job-post-mortem/24841 and https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839

it is also consistent with the previous exec dealing with LITE-PSM-USDC-A https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20July%2025%2C%202024.md#lite-psm-usdc-a-phase-1-actions

### Update PSM-USDC-A Fees

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

> If this executive proposal passes, then PSM-USDC-A fees will be updated as follows:

Reviewer: ```suggestion
If this executive proposal passes, then the PSM-USDC-A fees will be updated as follows:
```

Reviewer 2: see above

Reviewer: "the" refers to the fees in this sentence.

Reviewer 2: Understood

- PSM-USDC-A [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) will be increased by 0.01 percentage points from 0% to **0.01%**.
- PSM-USDC-A [Fee Out (`tout`)](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) will be increased by 0.01 percentage points from 0% to **0.01%**.

### Update PSM-USDC-A DC-IAM Parameters

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then the PSM-USDC-A DC-IAM parameters below will be updated as follows:

> - PSM-USDC-DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be decreased by 7,500 million DAI from 10,000 million DAI to **2,500 million DAI**.

Reviewer 4: ```suggestion
- PSM-USDC-A DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be decreased by 7,500 million DAI from 10,000 million DAI to **2,500 million DAI**.
```

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then the MCD_LITE_PSM_USDC_A fixed-sized amount of pre-minted Dai will be increased as follows:

- MCD_LITE_PSM_USDC_A [Fixed-sized Amount of Pre-minted Dai (`buf`)](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512#lite-psm-usdc-a-parameters-overview-8) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.

### Update LITE-PSM-USDC-A DC-IAM Parameters

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, then the LITE-PSM-USDC-A DC-IAM will be updated as follows:

> - LITE-PSM-USDC-DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased by 7,450 million DAI from 50 million DAI to **7,500 million DAI**.

Reviewer 4: ```suggestion
- LITE-PSM-USDC-A DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased by 7,450 million DAI from 50 million DAI to **7,500 million DAI**.
```

- LITE-PSM-USDC-A DC-IAM [Target Available Debt (`gap`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.
- LITE-PSM-USDC-A DC-IAM [Ceiling Increase Cooldown (`ttl`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-) will remain unchanged at **12 hours**.

### Update GSM Delay

> - **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)

Reviewer 3: ```suggestion
- **Authorization**: [Governance Facilitator Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/3), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
```

Since this is a Governance Scope item, I believe it should be approved by a Governance Facilitator

- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

If this executive proposal passes, the [GSM Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be updated as follows:

- The GSM Pause Delay will be increased by 14 hours from 16 hours to **30 hours**.

### Update LitePSM Keeper Network Job

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)

> If this executive proposal passes, then the LitePSM Keeper Network Job will be updated with the following contract calls and parameters as a consequence of the LITE-PSM-USDC-A DC-IAM and `buf` parameter changes described earlier in this document:

Reviewer: ```suggestion
If this executive proposal passes, then the LitePSM Keeper Network Job will be updated with the following contract calls and parameters, as a consequence of the LITE-PSM-USDC-A DC-IAM and `buf` parameter changes described earlier in this document:
```

- The old LitePSM job will be removed from the CronSequencer through contract call `sequencer.removejob(0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be)`.
- The new LitePSM job will be added to the CronSequencer through contract call `sequencer.addjob(0x0c86162ba3e507592fc8282b07cf18c7f902c401)`.
- [`fill`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [rushThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be set to **20 million DAI**.
- [`trim`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [gushThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be set to **20 million DAI**.

> - [`chug`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [cutThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be remain unchanged at **300,000 DAI**.

Reviewer: ```suggestion
- [`chug`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [cutThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will remain unchanged at **300,000 DAI**.
```

- The chainlog entry for [CRON_LITE_PSM_JOB](https://etherscan.io/address/0x689ce517a4dfcf0c5ec466f2757d324fc292c8be) will be updated to [0x0c86162ba3e507592fc8282b07cf18c7f902c401](https://etherscan.io/address/0x0c86162ba3e507592fc8282b07cf18c7f902c401).

The new LitePSM job meets all the requirements outlined in the [LitePSM Job Post-Mortem](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841).

## Review

Community debate on these topics can be found on the MakerDAO [Governance forum](https://forum.makerdao.com/). Please review any linked threads to inform your position before voting.

---

## Resources

> 

Reviewer: For more information on the LitePSM (LITE-PSM-USDC-A), please refer to the [LitePSM (LITE-PSM-USDC-A): Introduction and Overview](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)
